MetamodelAlgemeen.md

```diff
@@ -198,16 +199,47 @@ Diagram: Keuze tussen attribuutsoorten binnen een attribuutsoort. Patroon geldt
 
 *Use case 4: Keuze tussen relatiedoelen, als nadere invulling van 1 betekenisvolle relatiesoort*
 
-Dit diagram is uitgewerkt voor Objecttype. Voor Gegevensgroeptype geldt hetzelfde patroon, behalve dat een Gegevensgroeptype geen doel mag zijn van een Relatiesoort.
+Deze use case komt in de praktijk weinig voor, maar het kan wel, daarom is deze opgenomen. 
 
-Een objecttype of gegevensgroep kan normaal een relatiesoort hebben, die gekoppeld is aan een objecttype. Als er sprake is van een relatiesoort met een keuze, dan is het relatiedoel van de relatiesoort niet gekoppeld aan 1 objecttype, maar dan is het objecttype gekoppeld aan een keuze en deze keuze geeft 2 of meer relatiedoelen aan.
+Het bron object heeft 1 relatie, en heeft 1 gerelateerd doel object, maar dit doel object kan van verschillende objecttypes zijn. Het bron objecttype kent slechts 1 relatiesoort/relatierol, met een eigen betekenis, en deze relatiesoort/rol verwijst naar 2 of meer objecttypes. 
+
+Dit kan voorkomen wanneer vanuit het bron objecttype gezien er maar sprake is van 1 concern dat ingevuld moet worden, ongeacht welk objecttype dit concern zal gaan invullen. 
+
+Dit diagram is uitgewerkt voor Objecttype. Voor Gegevensgroeptype geldt hetzelfde patroon (1 relatiesoort vanuit een gegevensgroeptype naar een objecttype). 
 
 <figure id="KeuzeRelatiedoel4">
   <img src="media/KeuzeRelatiedoel4.png" alt="" />
   <figcaption>Keuze tussen relatiedoelen</figcaption>
 </figure>
 
-Diagram: Keuze tussen relatiedoelen. Patroon geldt ook voor Gegevensgroeptype, behalve dat een Gegevensgroeptype geen doel mag zijn van een Relatiesoort.
+Diagram: Keuze tussen relatiedoelen. 
+
+Uitwerking met een keuze constructie: een bron objecttype of gegevensgroeptype kan normaal een relatiesoort/relatierol hebben, die gekoppeld is aan (verwijst naar) een doel objecttype. Als er sprake is van een relatiesoort/relatierol met een relatiedoel keuze, dan is de relatiesoort niet gekoppeld aan 1 objecttype, maar dan is de relatiesoort gekoppeld aan een keuze en deze keuze is gekoppeld aan 2 of meer doel objecttypen. Elk objecttype is een afzonderlijk relatiedoel. Er moet voor deze ene relatiesoort/relatierol een keuze gemaakt worden voor 1 van de relatiedoelen. Het gerelateerde object is 1 van deze objecttypen, maar de betekenis van de relatie is hetzelfde, ongeacht van welk objecttype het object is. 
+
+Opmerkingen: 
+- de relatiesoort kan een eigen relatienaam hebben en elk relatiedoel kan een eigen rolnaam kan hebben
```

Comment:
Dit is zelfs aan te bevelen omdat een Keuze bijna altijd lijdt tot een als-dan constructie in de code. Met een naam van de relatie kun je dat makkelijker maken als "A dan x als B dan y doen". Deze naam moet dan ook in de output terecht komen en niet worden weggepoetst.